src/transformer/events/mod_quiz/question_answered/match.php

```diff
@@ -60,7 +60,7 @@ function ($reduction, $selection) {
         ],
         'timestamp' => utils\get_event_timestamp($event),
         'result' => [
-            'response' => $questionattempt->responsesummary,
+            'response' => ($questionattempt->responsesummary ? $questionattempt->responsesummary:''),
```

Comment:
This could be simplified a bit by using `?:`.

```suggestion
            'response' => ($questionattempt->responsesummary ?: ''),
```

Reply:
OK - made this change